CHANGELOG.md

```diff
@@ -1,127 +1,32 @@
+* TBD
 
 ## Breaking Changes
 
+* TBD
+
 The following classes, methods, properties and events have been deprecated for at least 2 releases and have now been removed from the SDK:
 
 | Class/Property/Method/Event | Alternate option | Version deprecated |
 |----------|-------------|--------------------|
-| `esri/views/layers/support/FeatureEffect` | Use [esri/layers/support/FeatureEffect](https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-support-FeatureEffect.html) instead | 4.22 |
-| `esri/views/layers/support/FeatureFilter` | Use [esri/layers/support/FeatureFilter](https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-support-FeatureFilter.html) instead | 4.22 |
-| `BasemapToggle.toggle` and `BasemapToggleViewModel.toggle` events | Watch the [activeBasemap](https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-BasemapToggle.html#activeBasemap) property instead | 4.22 |
-| `CSVLayerView.effect` | Use [CSVLayerView.featureEffect](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-layers-CSVLayerView.html#featureEffect) instead | 4.22 |
-| `FeatureLayerView.effect` | Use [FeatureLayerView.featureEffect](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-layers-FeatureLayerView.html#featureEffect) instead | 4.22 |
-| `GeoJSONLayerView.effect` | Use [GeoJSONLayerView.featureEffect](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-layers-GeoJSONLayerView.html#featureEffect) instead | 4.22 |
-| `OGCFeatureLayerView.effect` | Use [OGCFeatureLayerView.featureEffect](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-layers-OGCFeatureLayerView.html#featureEffect) instead | 4.22 |
-| `PrintViewModel.scaleEnabled` | Use [TemplateOptions.scaleEnabled](https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Print-TemplateOptions.html#scaleEnabled) or [PrintTemplate.scalePreserved](https://developers.arcgis.com/javascript/latest/api-reference/esri-rest-support-PrintTemplate.html#scalePreserved) instead | 4.22 |
-| `SearchViewModel.defaultSymbol` | Use [SearchViewModel.defaultSymbols](https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Search-SearchViewModel.html#defaultSymbols) instead | 4.22 |
-| `StreamLayerView.effect` | Use [StreamLayerView.featureEffect](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-layers-StreamLayerView.html#featureEffect) instead | 4.22 |
-| `WFSLayerView.effect` | Use [WFSLayerView.featureEffect](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-layers-WFSLayerView.html#featureEffect) instead | 4.22 |
+| `esri/widgets/Slice/SlicePlane` | Use [esri/analysis/SlicePlane](https://developers.arcgis.com/javascript/latest/api-reference/esri-analysis-SlicePlane.html) instead | 4.23 |
```

Comment:
I think this whole table can be deleted until there are breaking changes introduced.

Reply:
👍